<?xml version="1.0" encoding="UTF-8"?>
<CustomObjectTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
    <layouts>
        <layout>Error Layout</layout>
        <sections>
            <label>Links personalizados</label>
            <section>Custom Links</section>
        </sections>
    </layouts>
    <nameFieldLabel>Número do erro</nameFieldLabel>
    <standardFields>
        <name>Name</name>
        <label>Número do erro</label>
    </standardFields>
    <gender>Masculine</gender>
    <caseValues>
        <plural>false</plural>
        <value>Erro</value>
    </caseValues>
    <caseValues>
        <plural>true</plural>
        <value>Erros</value>
    </caseValues>
    <fields>
        <name>Context_Type__c</name>
        <help>Contexto que gerou o erro, se conhecido.</help>
        <label>Tipo de contexto</label>
    </fields>
    <fields>
        <name>Datetime__c</name>
        <help>Data e hora em que ocorreu o erro</help>
        <label>Data e hora</label>
    </fields>
    <fields>
        <name>Email_Sent__c</name>
        <help>Indica se uma notificação por email foi enviada a respeito desse erro.</help>
        <label>Email enviado</label>
    </fields>
    <fields>
        <name>Error_Type__c</name>
        <help>Tipo de erro ocorrido</help>
        <label>Tipo de erro</label>
    </fields>
    <fields>
        <name>Full_Message__c</name>
        <help>Texto completo da mensagem de erro</help>
        <label>Mensagem completa</label>
    </fields>
    <fields>
        <name>Object_Type__c</name>
        <help>Tipo de objeto no qual ocorreu o erro, se conhecido.</help>
        <label>Tipo de objeto</label>
    </fields>
    <fields>
        <name>Posted_in_Chatter__c</name>
        <help>Indica se o erro foi postado no Chatter.</help>
        <label>Postado no Chatter</label>
    </fields>
    <fields>
        <name>Record_URL__c</name>
        <help>Um link para o registro que causou o erro, se disponível.</help>
        <label>URL do registro</label>
    </fields>
    <fields>
        <name>Related_Record_ID__c</name>
        <help>O ID do Salesforce do registro associado a este erro.</help>
        <label>ID do registro relacionado</label>
    </fields>
    <fields>
        <name>Retry_Pending__c</name>
        <help>Quando marcado, o registro especificado no ID do registro rel. será reprocess. no próximo processo agendado. Se o reprocess. for bem-sucedido, este campo será então desmarcado. Aplica-se a tipos específicos de erros que ocorrem nos processos agendados.</help>
        <label>Nova tentativa pendente?</label>
    </fields>
    <fields>
        <name>Stack_Trace__c</name>
        <help>Vestígio de pilha do erro lançado, se disponível no tempo de execução.</help>
        <label>Vestígio de pilha</label>
    </fields>
</CustomObjectTranslation>
